hypothesisfuzzer/fuzzing/repo_fuzzer.py

```diff
@@ -60,13 +59,18 @@ def get_errors(self):
             return jsonify(json.load(file_data))
 
     def _clone_git(self, git_url):
-        # Delete old code folder
 
+        # Pull repository if already exists
         if os.path.exists(self.name):
```

Comment:
Also should check that it is indeed a git repo and not just some random folder with that name.

Reply:
@Multramate